### ISSUE TITLE
Ability to serialise JSON without escaping forward slashes

### ISSUE THREAD

**Issue by IvanRibakov** [oatpp/oatpp#381]
According to the description at https://www.json.org/json-en.html, unescaped forward slashes (solidus) are legal characters in the JSON value sequence. Yet Oatpp chooses to escape them. Is there a way to disable forward slash escaping?

**Comment by IvanRibakov** [comment]
![image](https://user-images.githubusercontent.com/3639958/104926751-d4841f00-59a0-11eb-9bb9-3d5acb8a8d70.png)

Notice the

> Any codepoint except `"` or `\` or control characters

block that is an alternative to the "escaping path". Forward slash satisfies that criteria and hence is NOT mandatory to be escaped. 

If you would like a more formal confirmation of this than an interpretation of some diagram, the same link contains following definition of JSON character:

```
character
    '0020' . '10FFFF' - '"' - '\'
    '\' escape
```

which should be interpreted as follows - character is either:
- a symbol in the range `0020` - `10FFFF` (except `"` and `\`)
- or a `\` followed by one of the symbols from the `escape` collection. 

Notice that forward slash's codepoint is `002F` and it perfectly satisfies first definition of a `character`

> You can also play with an online JSON string formatted - https://www.freeformatter.com/json-escape.html

Just because some tools choose to force the escaping of all characters (kinda makes sense since the link you provided advertises to be demonstrating just that) doesn't mean that it is mandatory, necessary or even desired in all cases.

**Comment by lganzzzo** [comment]
@IvanRibakov ,

I've double-checked - yes, you are correct.
We can add this option to the serializer config.

What is your use-case, why don't you want to escape `/` - char? (except that it's unnecessary :) )

**Comment by IvanRibakov** [comment]
We're using Oatpp serialiser for logging JSON objects. With forced escaping searching for some literal values will not produce expected results.

**Comment by lganzzzo** [comment]
Hey @IvanRibakov ,

Please check out this branch - `better_json`.

To disable solidus escaping use the following code:

```cpp
objectMapper->getSerializer()->getConfig()->escapeFlags &= !oatpp::parser::json::Utils::FLAG_ESCAPE_SOLIDUS;
```

Please let me know if it works for you.

Regards,
Leonid

**Comment by IvanRibakov** [comment]
HI Leonid,

unfortunately I'm unable to check the new feature by running it (time constraints), but the proposed solution seems reasonable and should allow us to achieve desired behaviour fairly easily.

Will keep an eye on the upcoming releases.

Thanks for a quick turnaround!

**Comment by lganzzzo** [comment]
Fixed in `1.3.0`, see [json-serializer-escape-flags](https://github.com/oatpp/oatpp/blob/master/changelog/1.3.0.md#json-serializer-escape-flags)
